src/test/crimson/seastore/nvmedevice/test_nvmedevice.cc

```diff
@@ -57,6 +57,7 @@ TEST_F(nvdev_test_t, write_and_verify_test)
   run_async([this] {
     device.reset(new random_block_device::nvme::NVMeBlockDevice(""));
```

Comment:
The correct device path should be passed to the constructor. And drop the external path parameter from open()